projects/control-service/cicd/deploy-testing-pipelines-service.sh

```diff
@@ -15,7 +15,7 @@ SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" >/dev/null 2>&1 && pwd)"
 
 export TAG=${TAG:-$(git rev-parse --short HEAD)}
 export FRONTEND_TAG=${FRONTEND_TAG:-$(git rev-parse --short HEAD)}
-export RELEASE_NAME=${RELEASE_NAME:-cicd-control-service}
+export RELEASE_NAME=${RELEASE_NAME:-cicd-vdk}
```

Comment:
could we maybe do vdk-cs instead of vdk

Reply:
Actually having cs in the name is bad. 
the helm chart encompases much more than the control service. 
It is the UI and database etc. so I don't think it should have cs in the name
WDYT?

Reply:
@murphp15 I agree it should not be cs, but vdk does nor seem to be a good name, too. I would go for cicd-all or something like this that is more informative of what is releasing

Reply:
but then you have no way of knowing it is the helm for VDK?
do you mean cicd-vdk-all?